.github/workflows/build-test.yml

```diff
@@ -192,6 +192,67 @@ jobs:
       - name: Check dependencies
         run: npm run checkDependencies
 
+  vsc_api_check:
```

Comment:
Created a separate Job that will test compilation against the latest type definition files.
This way we can have CI running and perfectly green until VS Code actually ships stuff.

Reply:
For instance, I should have be able to run CI right now with the current VSC Insiders.

Reply:
This will allow us to have both